bayes_opt/observer.py

```diff
@@ -29,20 +35,19 @@ def _update_tracker(self, event, instance):
         """
         if event == Events.OPTIMIZATION_STEP:
             self._iterations += 1
-            
+
             if instance.max is None:
                 return
 
             current_max = instance.max
 
-            if (self._previous_max is None
-                    or current_max["target"] > self._previous_max):
+            if self._previous_max is None or current_max["target"] > self._previous_max:
                 self._previous_max = current_max["target"]
                 self._previous_max_params = current_max["params"]
 
     def _time_metrics(self):
         """Return time passed since last call."""
-        now = datetime.now()
+        now = datetime.now(_UTC)
```

Comment:
I'm not sure I'm reading this correctly -- you're setting `now` to the UTC time? I think that's fine if we just use timedelta (i.e. never explicitly print the time) but otherwise we should probably use the timezone of the user's system.

Reply:
The `ruff` convention requires to specify `tzinfo`
when using the `.now()` method, so I applied `UTC`.

I decided on `UTC` because getting the exact local timezone is a bit tricky.
(although it would be simple to add other library).
I'll remove the rule from this part.